data.yml

```diff
@@ -75,6 +75,11 @@ landscape:
       - subcategory:
         name: Observability & Evaluation
         items:
+          - item:
```

Comment:
Would it be possible to maintain the order? We prioritize open-source products at the top. Thanks!

Reply:
Of course, sorry for the confusion. Will submit another PR, thanks!

Reply:
@gaocegege could you please review and approve this PR? Have prioritised open-source products at the top and added our crunchbase! Thanks :)